package.json

```
@@ -186,6 +186,7 @@
     "react-dom-16": "npm:react-dom@16.14.0",
     "react-dom-17": "npm:react-dom@17.0.2",
     "react-virtualized": "9.22.3",
+    "react-virtuoso": "^2.17.2",
```

With the current config off renovate dev-depedencies are pinned :slightly_smiling_face: 
```suggestion
    "react-virtuoso": "2.17.2",
```

---

Good catch (again) 👍